models/course.js

```diff
@@ -0,0 +1,85 @@
+const { connector } = require("#models/databaseUtil");
+
+const courseSchema = {
+  name: { type: String, required: true },
+  code: { type: String, required: true },
+  theoryHours: { type: Number, required: true },
+  tutorialHours: { type: Number, required: true },
+  practicalHours: { type: Number, required: true },
+  totalHours: {   // virtual to return total hours based on th + tut + prac hrs
```

Comment:
refer to this for creating virtual attribute
https://mongoosejs.com/docs/tutorials/virtuals.html#virtuals-via-schema-options